LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ExtraRules.swift
+//  LivrCommonCustomRules
+//
+//  Created by Felipe Marino on 03/07/19.
+//  Copyright © 2019 Zap SA Internet. All rights reserved.
+//
+
+import Foundation
+import Livr
+
+struct CustomRules {
+    
+    struct GreaterThanOrEqualToField: LivrRule, PreDefinedRule, RuleThatCompareFields, CustomRule {
+        var name: String = "greater_eq_than_field"
```

Comment:
private? What do you think?

Reply:
It is already set as private in the protocol :). As `get` only.